Dockerfile

```
@@ -0,0 +1,46 @@
+        git \
+        sudo \
+        python3 \
+        python3-pip \
+        make
+
+# install dependencies into temp directory
+# this will cache them and speed up future builds
+FROM base AS install
+RUN mkdir -p /temp/dev
+COPY package.json bun.lock /temp/dev/
+RUN cd /temp/dev && bun install --frozen-lockfile
```

Hadolint (2.12.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)